source/clear-linux/get-started/virtual-machine-install/vmware-player.rst

```diff
@@ -1,116 +1,528 @@
 .. _vmware-player:
 
-Use VMware\* Player
-###################
+Run Clear Linux as a VMware\* Workstation Player guest OS
+#########################################################
 
-This section explains how to run Clear Linux OS for Intel® Architecture
-within a `VMware Player`_ environment.
+`VMware Workstation 14 Player`_ is a type 2 hypervisor which runs on top of another
+operating system such as Windows or Linux.  Creation, configuration, management, and execution of virtual
+machines are done on the local system.  For the cloud version of VMware, see 
+:ref:`vmware-esxi`.
 
-Please ensure you have enabled `Intel® Virtualization Technology
-<http://www.intel.com/content/www/us/en/virtualization/virtualization-technology/intel-virtualization-technology.html>`_
-(Intel® VT) and `Intel® Virtualization Technology for Directed I/O
-<https://software.intel.com/en-us/articles/intel-virtualization-technology-for-directed-io-vt-d-enhancing-intel-platforms-for-efficient-virtualization-of-io-devices>`_
-(Intel® VT-d) in your BIOS/UEFI firmware configuration.
+This section explains how to run |CLOSIA| as a virtual machine using the 
+VMware Workstation 14 Player hypervisor.
 
-Install VMware Player
-=====================
+There are 2 ways to create a |CL| :abbr:`VM (Virtual Machine)` to run in VMware:
 
-VMware Workstation Player, formerly VMware Player, is a virtualization
-software package for x64 computers running Microsoft Windows or Linux.
-Download VMware player from the `VMware website`_.
+* `Method #1`: Install |CL| into a new VM.  This provides flexibility 
+  in configuring the VM size, partitions, initial |CL| bundles selection, etc.
+* `Method #2`: Use a ready-made VMware |CL| image.  
```

Comment:
Confusing: replace ready-made with pre-configured or supplied and link to the downloads page.